spec/welcome/home_page_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe "Welcome Page" do
+  it "should have landing page, log in link and todays neos" do
+    visit "/"
+
+    expect(page).to have_content("WELCOME TO HORROR SCOPES")
+    expect(page).to have_content("TODAYS TRAGEDY AS PREDICTED BY THE PLANETS")
+    expect(page).to have_content("LOG IN")
+    expect(page).to have_content("HOME")
+    expect(page).to have_css(".cathulu")
```

Comment:
can't figure out how to get the test to actually make sure the image is there. It should up on localhost but needs to be tested.